### ISSUE TITLE
Streaming not working with proxy after updating to SDK version 5.12.0

### ISSUE THREAD

**Issue by fredwes** [launchdarkly/node-server-sdk#186]
I am unable to successfully initialize the SDK (in streaming mode) after updating to version 5.12.0.

- Polling works
- Initialization (with streaming) works when not using a proxy

**Comment by eli-darkly** [comment]
It'd be easier for us to investigate this if you could provide some more information, as we request in the template that comes up when you start an issue:

* log output
* Node version
* OS

Also, could you please show how you are configuring the SDK (with any sensitive information such as SDK key x'd out)?

**Comment by eli-darkly** [comment]
Another helpful piece of information would be what SDK version you were using when it _did_ work. There are a lot more things that changed between 5.0.0 and 5.12.0 than between 5.11.3 and 5.12.0, for instance.

**Comment by eli-darkly** [comment]
I am able to reproduce a failure when using 5.12.0 with a proxy, and I know of at least one fix that is necessary, but I'd still like to get the above information to make sure that what you're seeing is really the same failure and not something else that also needs to be fixed.

**Comment by fredwes** [comment]
@eli-darkly Version 5.11.3 works and all versions 5.12.0 and greater do not.

- log output: `Received I/O error for streaming request - will retry`, `Will retry stream connection in 125 milliseconds`, etc.
- node version: 10.14.2
- OS: OSX

SDK configuration:
```
LaunchDarklySDK.init('sdk-key', {
    logger,
    proxyHost: 'proxy.com',
    proxyPort: 1234,
    allAttributesPrivate: true,
    stream: true,
    sendEvents: false
  });
```

**Comment by eli-darkly** [comment]
Thanks! Looks like this is a straightforward bug in 5.12.0 that was my fault - it happened when we switched the Node SDK to use some shared code for streaming, instead of its own copy of that code which had a change in it that I had missed. Sorry about this and we'll have a patch out ASAP.

**Comment by fredwes** [comment]
Thanks @eli-darkly !

**Comment by eli-darkly** [comment]
There's a fix for this in 5.13.1. Please let us know when you've had a chance to retest with this release.

**Comment by fredwes** [comment]
Confirmed working now with v5.13.1. Thanks again @eli-darkly !
